Automattic/wp-calypso
# Jetpack Onboarding: Setup initial structure

This PR provides the basic structure for the Jetpack Onboarding project:

* Provide a feature flag, enabled only on the `development` environment
* Allow `Wizard` steps to retrieve back/forward URL (addressed separately in #20402)
* Add a main wrapper component for the steps logic
* Define the steps and their components
* Create the Jetpack Onboarding section and its routes and controller.
* Add all steps as empty components with just title and subtitle.

Based on the following initial prototype PR: #20360

To test:
* Checkout this branch
* Go to http://calypso.localhost:3000/jetpack/onboarding/
* Go through back and forth through all steps and verify you can navigate properly.
* Try visiting each of the steps in a new window and verify you can access them.
